eslint.config.js

```diff
@@ -313,34 +308,18 @@ export default [
 			"dist-module/jquery.module.js",
 			"dist-module/jquery.slim.module.js",
 			"dist-module/jquery.factory.module.js",
-			"dist-module/jquery.factory.slim.module.js"
-		],
-
-		languageOptions: {
-			globals: {
-				...globals.es2021,
-				define: false,
-				module: false,
-				Symbol: false
-			}
-		}
-	},
-
-	{
-		files: [
+			"dist-module/jquery.factory.slim.module.js",
 			"dist/jquery.bundler-require-wrapper.js",
-			"dist/jquery.bundler-require-wrapper.slim.js",
-			"dist-module/jquery.node-module-wrapper.js",
-			"dist-module/jquery.node-module-wrapper.slim.js"
+			"dist/jquery.bundler-require-wrapper.slim.js"
 		],
 
 		languageOptions: {
+			ecmaVersion: 2015,
```

Comment:
Do we set `ecmaVersion` to `2015` here for all `dist` & `dist-module` files and then change it to `5` for `dist` files? Does entries below overwrite what's above for matching files?

It still feels a bit confusing to me. Could we avoid setting `2015` for `dist` at all?